tests/rule_based_profiler/parameter_builder/test_numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -326,13 +334,224 @@ def test_oneshot_numeric_metric_range_multi_batch_parameter_builder_bobby(
         ]
     )
 
+    # Assert no significant difference between expected (null hypothesis) and actual estimation histograms.
+    ks_result = stats.ks_2samp(
+        data1=actual_estimation_histogram, data2=expected_estimation_histogram
+    )
+    p_value: float = ks_result[1]
+    assert p_value > 9.5e-1
+
+
+def test_oneshot_numeric_metric_range_multi_batch_parameter_builder_with_evaluation_dependency_bobby(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    metric_domain_kwargs: dict = {"column": "fare_amount"}
+
+    fully_qualified_parameter_name_for_value: str = "$parameter.column_min_range"
+
+    expected_value_dict: dict
```

Comment:
Shouldn't this type hint be moved down to line 421 since there is only one assignment?

Reply:
@NathanFarmer I can look and apply -- in a quick follow up PR.